src/app/zone-egresses/views/IndexView.vue

```diff
@@ -8,23 +8,24 @@
       v-slot="{ route, t }"
       name="zone-egress-list-view"
       :params="{
-        page: 1,
-        size: me.pageSize
+        /* page: 1, */
+        /* size: me.pageSize, */
+        zone: ''
       }"
     >
       <AppView>
         <template #title>
-          <h1>
+          <h2>
             <RouteTitle
               :title="t('zone-egresses.routes.items.title')"
               :render="true"
             />
-          </h1>
+          </h2>
         </template>
 
         <DataSource
           v-slot="{ data, error }: ZoneEgressOverviewCollectionSource"
-          :src="`/zone-egress-overviews?page=${route.params.page}&size=${route.params.size}`"
+          :src="`/zone-cps/${route.params.zone || '*'}/egresses?page=${'1'}&size=${'100'}`"
```

Comment:
route.params.zone will be empty when zones are disabled (`zones/egresses` vs `zones/zone-cps/name-of-parent/egresses)` so this is where we use `*`/"gimme all of them"